go-controller/pkg/ovn/pods.go

```diff
@@ -251,7 +252,8 @@ func (oc *Controller) deleteLogicalPort(pod *kapi.Pod, portInfo *lpInfo) (err er
 		allOps = append(allOps, ops...)
 	}
 	ops, err = oc.delLSPOps(logicalPort, nodeName, portUUID)
-	if err != nil {
+	// Tolerate cases where logical switch of the logical port no longer exist in OVN.
+	if err != nil && !errors.Is(err, libovsdbclient.ErrNotFound) {
```

Comment:
this feels like dejavu that we have fixed this in many deletion places, but good catch here, I can imagine this must be causing infinite retry...